pipelines/tasks/Kallisto.wdl

```diff
@@ -0,0 +1,155 @@
+
+task Mkref {
+  File transcriptome_fasta  # fasta file containing transcripts ONLY (NOT a full genome!)
+  Int k  # size of kmer in index; must be an odd number.
+
+  command {
+    kallisto index \
+      -i kallisto.idx \
+      -k ${k} \
+      "${transcriptome_fasta}"
+  }
+
+  runtime {
+    docker: "humancellatlas/kallisto:0.43.1"
+    cpu: 4
+    memory: "16 GB"
+    disks: "local-disk 150 HDD"
+  }
+
+  output {
+    File index = "kallisto.idx"
+  }
+}
+
+
+task QuantSingleEnd {
+  File r1  # forward read
+  File index  # kallisto index, output of Kallisto.Mkref task
+
+  # estimate disk requirements
+  Int disks = 10 + ceil(size(r1, "G") * 4 + size(index, "G") * 2)
+
+  command {
+    kallisto quant \
+      --index "${index}" \
+      --output-dir . \
+      --bootstrap-samples 100 \
```

Comment:
I would add '--seed' to have fixed random number. it is just easier from testing point of view.

Reply:
Default of 42 should produce a fixed seed, no? Is there a point in setting it to something else that I'm missing?

Reply:
copy paste from manual "--seed=INT                Seed for the bootstrap sampling (default: 42)"
http://pachterlab.github.io/kallisto/manual.html